### DISCUSSION TITLE
UMAF problem in Alienware M18 r1 7845hx 4080

### DISCUSSION THREAD

**Discussion by SiegeSledge** [DavidS95/Smokeless_UMAF#48] [Q&A]
Reason of trying UMAF:This is a recording of me at the OW .The fps display is on top left corner.In past three days since I received the computer no matter what I do（nomatter gaming or just youtube） the frame rate randomly drops to 20 and then recovers within a second, once or twice an hour. Configuration is 7845HX,4080, AWCC performance mode doesn't work, installed all available driver updates and system updates, tried resetting the system to the outgoing version doesn't work, any solutions?
Then I learned about turning off some option like fTPM,adjust cpu offset,SAM,cppc and cppc pc .etc will fix the problem  but the original Alienware insyde bios does not provide these options . When I enter UMAF the device manager menu is empty,both first and second one.Thanks for any suggestions or solution for either of the problem.
![image](https://github.com/DavidS95/Smokeless_UMAF/assets/99249124/1925100c-faaa-4379-9a8a-bdf4968eca9c)
![image](https://github.com/DavidS95/Smokeless_UMAF/assets/99249124/043ed398-6591-403f-8659-5b8e00f41a0a)

**Comment by RandomLemon** [comment]
I have the same problem with my 7745HX Jiaolong MRID6.

**Comment by TLCYZ** [comment]
I think maybe alienware's bios banned the access of UMAF after bios version 1.2.2, however, i got some problems with falshing bios so i can only use the latest bios and thus facing the exact same issue right now, have u find out any solutions to this?

**Reply by PythonProgrammer77** [reply]
Were you able to find a solution to this? I've tried to downgrade bios versions a lot of different ways but have had no luck.

**Comment by PythonProgrammer77** [comment]
I have this exact same issue on my Alienware M16 R1 AMD. Just shows a blank list. Last commit was like a year ago, is an update coming for smokeless_umaf?
